src/extension/inpage-script/index.js

```diff
@@ -8,6 +9,9 @@ if (document) {
   if (!window.webln) {
     window.webln = new WebLNProvider();
   }
+  if (!window.alby) {
```

Comment:
Other inpage scripts are loaded in a separate file. Is there a reason `window.alby` is handled differently?

Reply:
no. thanks. did not think about that. 
changed it.